css-positioning/fixed.css

```diff
@@ -0,0 +1,12 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+
+  /* your code here */
+  position: fixed;
+  left: 200px;
```

Comment:
Rather than this manual movement, try making this element the full width of the viewport. It already has `text-align: center` applied, which will handle centering the text.